data/actionsmap/yunohost.yml

```diff
@@ -291,6 +291,17 @@ domain:
                     extra:
                         pattern: *pattern_domain
 
+        ### dns_conf()
+        dns-conf:
+            action_help: Generate DNS configuration for a domain
+            api: GET /domains/<domain>/dns
+            configuration:
+                lock: false
+                authenticate: all
```

Comment:
One more thing - which is not documented too ;) - regarding `authenticate: all`, it will imply the admin password to be asked in the cli too - which is maybe too much for this action... You can omit it and remove the first `auth` argument in the function - which is passed only if `authenticate` is `all`. It will result in a required authentication just for the api - which is the default: https://github.com/YunoHost/yunohost/blob/unstable/data/actionsmap/yunohost.yml#L38. I know, it's quite dirty, that's why a rewriting is needed!